src/pages/home/report/ContextMenu/ReportActionContextMenu.ts

```diff
@@ -123,33 +123,37 @@ function showContextMenu(
     if (!contextMenuRef.current) {
         return;
     }
+    const show = () => {
```

Comment:
I'm not sure how to give it an appropriate name, but 'show' seems to lack sufficient information to me 🥶

Reply:
I think it's fine because the function is declared within the `showContextMenu` body and the function itself is only for reusability, so there is nothing really much to be explained from its name.